samwise.js

```diff
@@ -2,9 +2,10 @@
  * Autoplay plugin - Automatically advance slideshow after N seconds
```

Comment:
SCRIPT-This function might benefit from using async/await instead of callbacks for better readability.